# Curriculum Vitae

 [ [About Me](/index.md) ][ **[Curriculum Vitae](/cv.md)** ][ [Contact](/contact.md) ][ [![ORCID iD icon](https://orcid.org/sites/default/files/images/orcid_16x16.png) ORCID](https://orcid.org/0000-0003-3192-6538) ][ [<img src="/assets/GitHub-Mark-32px.png" height="16" width="16"> Github](https://github.com/molikd) ] 


<!-- <table style="width:500px">
  <tr>
<td>[ <a href="/">About Me</a> ]</td>
<td>[ <strong><a href="/cv.html">Curriculum Vitae</a></strong> ]</td>
<td>[ <a href="/contact.html">Contact</a> ]</td>
<td>[ <a href="https://orcid.org/0000-0003-3192-6538">
<img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" alt="ORCID iD icon" /> ORCID</a> ]</td>
<td>[ <a href="https://github.com/molikd"><img src="/assets/GitHub-Mark-32px.png" height="16" width="16" /> Github</a> ]</td>
  </tr>
<table> -->


---
> # David C. Molik
> _Computational Ecology and Evolution PhD_
>
> ## EDUCATION
>
> **University of Notre Dame, Notre Dame, IN**  
> Phd. Integrated BioMedical Sciences  
> Concentration in Computational Biology   
> Advisor: [Michael Pfrender](https://www3.nd.edu/~mpfrende/)    
> Aug/2016 â€“ May/2021
>
> **Rensselaer Polytechnic Institute, Troy, NY**  
> BS. Information Technology and Web Science  
> Concentration in Communications  
> Aug/2009 â€“ Dec/2012  
> 
> ## EMPLOYMENT  
>
> ### United States Department of Agriculture
>
> - **[current] Computational Biologist at [Arhtropod-borne Animal Disease Research Unit](https://www.ars.usda.gov/plains-area/mhk/cgahr/abadru/)**
    - Mar/2023 - 
> - **Research Biologist/PostDoc at [Tropical Pest Genetics and Molecular Biology Research Unit](https://www.ars.usda.gov/people-locations/people-list-offices/?modeCode=20-40-05-15)** 
>   - Jun/2021 â€“ Mar/2023
>
> ### Phd. Student Positions at the University of Notre Dame â€“ Notre Dame, IN
> 
> - **Graduate Scholar in [Navari Center for Digital Scholarship](https://cds.library.nd.edu/)** 
>   - Jan/2020 â€“ Jun/2021
> - **Graduate Research Assistant in collaboration with [Goodson lab](https://chemistry.nd.edu/people/holly-goodson/)**  
>   - May/2019 â€“ Aug/2019
> - **Graduate Research Assistant in collaboration with [Genomics and Bioinformatics Core](https://genomics.nd.edu/)**  
>   - May/2018 â€“ Aug/2018
> - **Graduate Research Assistant in collaboration with [Emrich lab](http://web.eecs.utk.edu/~semrich/)**  
>   - Aug/2016 â€“ Dec/2017
>
> ### Positions held prior to University of Notre Dame Phd. studentship
>
> - **Scientific Informatics Developer, [Cold Spring Harbor Labratory](https://www.cshl.edu/research/cancer/bioinformatics/)** â€“ Cold Spring Harbor,  NY  
>   - Dec/2014 â€“ Aug/2016  
> - **Software Engineer; Network Operations Center Manager, [Synacor](https://synacor.com/)** â€“ Buffalo, NY  
>   - July/2013 â€“ Nov/2014  
>
> ### Positions held during BS. studentship at Rensselaer Polytechnic Institute â€“ Troy, NY
>  
> - **Undergraduate Researcher, [Tetherless World Constellation](https://tw.rpi.edu//)** 
>   - Aug/2012 â€“ Jan/2013
> - **NSF REU, SUNY at Buffalo in [Corso Lab](http://web.eecs.umich.edu/~jjcorso/)**
>   - May/2012 â€“ Aug/2012
> - **Internship, [Tetherless World Constellation](https://tw.rpi.edu//)** 
>   - May/2010 â€“ Aug/2010
>  
> ## ARTICLES
>
> **[2024]**
>
> **[Submitted]** **[editorial]** Cannon, E., D.C. Molik, A. Wright, H. Zhang, L. Honaas, K. Chougule, and S. Dyer. â€œGuidelines for Gene and Genome Assembly Nomenclatureâ€ 
>
> **[Submitted]** Drolet, B., L. Reister-Hendricks, C. Mayo, C. Rodgers, D. Molik, D. McVey â€œIncreased virulence of Culicoides midge cell-derived blue-tongue virus in IFNAR miceâ€
>
> Wright, A., M. D. Wilkinson, C. Mungall, S. Cain, S. Richards, P. Sternberg, E. Provin, J. Jonathan, S. Geib, R. Daniela, K. Yook, L. Stein, D. Molik. "FAIR Header Reference genome: A TRUSTworthy standard." Briefings in Bioinformatics, 2024. [doi:10.1093/bib/bbae122](https://doi.org/10.1093/bib/bbae122)
>
> **[In Press]** Molik, David, Amanda R. Stahlke, Sharu P. Sharma, Tyler J. Simmonds, Renee L. Corpuz, Angela N. Kauwe, Jeremy E. Schrader, Charles J. Mason, Sheina B. Sim, and Scott M. Geib. "otb: Creating a HiC/HiFi Pipeline to Assemble the Prosapia bicincta Genome."  G3: Genes, Genomes, Genetics, 2024. [doi:10.1093/g3journal/jkae129](https://doi.org/10.1093/g3journal/jkae129)
>
> **[2023]**
>
> Scroggs SLP, Bird EJ, Molik DC, Nayduch D. â€œVesicular Stomatitis Virus Elicits Early Transcriptome Response in Culicoides sonorensis Cellsâ€ Viruses. 2023, [doi:10.3390/v15102108](https://doi.org/10.3390/v15102108)
>
> **[2022]**
>
> **[forum]** David C. Molik. "An Outsiderâ€™s Perspective on Why We Climb Mountains and Why Projects Like the i5k Matter" Journal of Insect Science, 2022. [doi:10.1093/jisesa/ieac038](https://doi.org/10.1093/jisesa/ieac038)
>
> David C. Molik. "ðš–ðšŽðš ðšŸðŸ·: expanding on old estimations of biodiversity from eDNA with a new database framework" Database, 2022. [doi:10.1093/database/baac032](https://doi.org/10.1093/database/baac032)
>
> **[2021]**
> 
> David C. Molik, DeAndre Tomlinson, Shane Davitt, Eric L. Morgan, Matt Sisk, Benjamin Roche, Natalie Meyers, Michael E. Pfrender. "Combining Natural Language Processing and Metabarcoding to Reveal Pathogen-Environment Associations" PLoS Negl Trop Dis, 2021. [doi:10.1371/journal.pntd.0008755](https://doi.org/10.1371/journal.pntd.0008755), [Kudos Website](https://link.growkudos.com/1ca143loef4)
>
>  Ong Joseph Y., Julia T Pence, David C Molik, Holly V Goodson. "Yeast grown in continuous culture systems can detect mutagens with improved sensitivity relative to the Ames test" PLoS ONE, 2021. [doi:10.1371/journal.pone.0235303](https://doi.org/10.1371/journal.pone.0235303)
>
> **[2020]**
>
> Molik, David C., Michael Pfrender, Scott Emrich. â€œUncovering Effects from the Structure of MetabarcodeSequences for Metagenetic and Microbiome Analysisâ€ Methods and Protocols, 2020. [doi:10.3390/mps3010022](https://doi.org/10.3390/mps3010022)
>
> **[2019]**
>
> æŽè¯£è¿œ *(Yiyuan Li)*, David Molik, Michael Pfrender. åŸºäºŽNextflowæž„å»ºçš„å®æ¡å½¢ç è‡ªåŠ¨åŒ–åˆ†æžæµç¨‹EPPSï¼ˆåŸºå› ç»„å¤šæ ·æ€§ä¸“åˆŠï¼‰*(EPPS, a metabarcoding bioinformatics pipeline using Nextflow)* . Biodiversity Science, 2019. [doi:10.17520/biods.2018211](http://www.biodiversity-science.net/CN/10.17520/biods.2018211).
>
> **[2018]**
>
> Ho, Yu-Jui, Naishitha Anaparthy, David Molik, Grinu Mathew, Toby Aicher, Ami Patel, James Hicks, and Molly Gale Hammell. "Single-cell RNA-seq analysis identifies markers of resistance to targeted BRAF inhibitors in melanoma cell populations." Genome research, 2018, 1353-1363. [doi:10.1101/gr.234062.117](https://doi.org/10.1101/gr.234062.117). 
>
> **[2016]**
>
> Cumming, Grant P., Edward Morris, Paul Simpson, Tara French, Eva Kahana, Joanne S. Luciano, and David Molik. "The future of post-reproductive healthâ€“The role of the Internet, the Web, information provision and access." Post Reproductive Health, 2016, 123-130. [doi:10.1177/2053369116647858](https://doi.org/10.1177/2053369116647858).
>
> ## Conference Papers and Reports
>
> **[2022]**
>
>  Walker, Kimberly. Divya Kalra, Rebecca Lowdon, Guangyi Chen, David Molik, Daniela C. Soto, Fawaz Dabbaghie, et al. "The third international hackathon for applying insights into large-scale genomic composition to use cases in a wide range of organisms" F1000Research, 2022. [doi:10.12688/f1000research.110194.1](https://doi.org/10.12688/f1000research.110194.1)
>
> **[2018]**
>
> Molik, David C., Michael Pfrender, Scott Emrich. "Effects from structure of Metabarcode Sequences on Lossy Analysis of Microbiome Data" IEEE International Conference on Bioinformatics and Biomedicine, Data Analytics in Metagenomics, 2018. [doi:10.1109/BIBM.2018.8621143](https://doi.org/10.1109/BIBM.2018.8621143).
>
> **[short]** Tomlinson, DeAndre. David Molik, Michael Pfrender, Scott Emrich. "The Effects of Normalization, Transformation, and Rarefaction on Clustering of OTU Abundance" IEEE International Conference on Bioinformatics and Biomedicine, 2018. [10.1109/BIBM.2018.8621347](https://doi.org/10.1109/BIBM.2018.8621347).
>
> **[2012]**
>
> **[short]** Molik, David, and Michelle Sharer. â€œThe Use of Recursive Bisection in the Mapping of Sample New York State Electoral Districts.â€ Proceedings of the 6th International Conference on Electronic Governance, 2012, 520â€“21. [doi:10.1145/2463728.2463844](https://doi.org/10.1145/2463728.2463844).
>
> ## Chapters and Whitepapers
>
> **[2020]**
>
> RDA COVID-19 Working Group. recommendations and guidelines. Research Data Alliance. 2020. [doi:10.15497/rda00046](https://doi.org/10.15497/rda00046).
>
> **[2017]**
>
> Cumming, Grant P., Tara French, Jamie Hogg, Douglas McKendrick, Heidi Gilstad, David Molik, and Joanne S. Luciano. "Trust and Provenance in Communication to eHealth Consumers." In The Digitization of Healthcare, pp. 189-203. Palgrave Macmillan, London, 2017. [doi:10.1057/978-1-349-95173-4_11](https://doi.org/10.1057/978-1-349-95173-4_11).
>
> Cumming, Grant P., Douglas McKendrick, Jamie Hogg, Tara French, Eva Kahana, David Molik, and Joanne S. Luciano. "Formulating eHealth Utilizing an Ecological Understanding." In The Digitization of Healthcare, pp. 81-98. Palgrave Macmillan, London, 2017. [doi:10.1057/978-1-349-95173-4_5](https://doi.org/10.1057/978-1-349-95173-4_5).
>
> ## ABSTRACTS - TALKS
> *Only Talks and Posters in which I am the First or Last Author*
>
> **[2024]**
>
> **[invited panel]** â€œData Science and Analytics Seminar Series - 'Computational Genomics, Data Science and AI Panelâ€™.â€ SUNY Buffalo State University Data Science and Analytics Seminar Spring 2024 Series. Buffalo, 2024. [dataanalytics.buffalostate.edu/dsa-seminar-series](https://dataanalytics.buffalostate.edu/dsa-seminar-series)
>
> **[talk]** Molik, David, Katheryn Buble, Leyla Cabugos, Jaqueline Campbell, Jenna Daenzer, Daniela Raciti, Leonore Reiser, Sook Jung, Adam Wright, Karen Yook. â€œFAIR Scientific Literature Is Not What You Think: How to Know Where Data Goes.â€ Plant and Animal Genomes 31, San Diego, 2024. [plan.core-apps.com/pag_2024/event/b44b10371958e4ecc7a77988eb6d15b7](https://plan.core-apps.com/pag_2024/event/b44b10371958e4ecc7a77988eb6d15b7)
>
> **[talk]** Molik, David, Joaquin Carbonara, Scott Geib, Marc Halfon, Olga S. Novikova, Jimmy Pitzer, Brian E. Scheffler, Phillip Shults, Tyler J. Simmonds, Caitlin Taylor. â€œMaking Culicoides sonorensis a Blueprint for Pest Management Model Systemsâ€ Plant and Animal Genomes 31, San Diego, 2024. [plan.core-apps.com/pag_2024/abstract/0576097b-5f81-45e2-bce6-df69d943ca9c](https://plan.core-apps.com/pag_2024/abstract/0576097b-5f81-45e2-bce6-df69d943ca9c)
>
> **[2023]**
>
> **[poster]** Wright, Adam, David Molik. â€œExpanding FAIR-BioHeaders for Comprehensive Data Management.â€ Genome Informatics, Cold Spring Harbor, 2023. [meetings.cshl.edu/meetings.aspx?meet=info&year=23](https://meetings.cshl.edu/meetings.aspx?meet=info&year=23)
>
> **[poster]** Molik, David, Amanda Stahlke. â€œOTB (Only The Best genome assembly tools)â€”A phased genome assembly nextflow pipeline.â€ Genome Informatics. Cold Spring Harbor, 2023. [meetings.cshl.edu/meetings.aspx?meet=info&year=23](https://meetings.cshl.edu/meetings.aspx?meet=info&year=23)
>
> **[invited panel]** â€œAI Across Academia, Industry, Government, and Societyâ€ SUNY Buffalo State University Data Science and Analytics Seminar Fall 2023 Series. Buffalo, 2023. [dataanalytics.  buffalostate.edu/fall-2023-seminar-series](https://dataanalytics.buffalostate.edu/fall-2023-seminar-series)
>
> **[2022]**
>
> **[poster]** Molik, David. â€œOTB: (Only The Best Genome Assembly Tools) Overcoming Scientific IT Infrastructure Problems for the Creation, Storage, and Use of a Large Number of Phased Insect Genomes: A Case Study Using The Two-Lined Spittlebugâ€ Arthropod Genomics Symposium. Notre Dame, 2022. [web.cvent.com/event/c19bd7ed-9ebc-47f4-9876-a524061871ed/summary](https://web.cvent.com/event/c19bd7ed-9ebc-47f4-9876-a524061871ed/summary)
>
> **[2020]**
>
> **[poster]** Molik, David, DeAndre Tomlinson, Shane Davitt, Eric L. Morgan, Benjamin Roche, Natalie Meyers, Michael E. Pfrender. "Automated text categorization and metabarcoding reinforces Cryptococcus neoformansâ€“woody decomposition association." Biological Data Science (Virtual). Cold Spring Harbor, 2020. [meetings.cshl.edu/meetings.aspx?meet=data&year=20](https://meetings.cshl.edu/meetings.aspx?meet=data&year=20)
>
> **[2019]**
>
> **[invited/talk]** Molik, David, Stephen Richards, Michael Pfrender. "Data mining for the prioritization of American insecta genomes." Entomology 2019. Entomological Society of America, 2019. [esa.confex.com/esa/2019/meetingapp.cgi/Paper/143297](https://esa.confex.com/esa/2019/meetingapp.cgi/Paper/143297).
>
> **[lightning talk/poster]** Molik, David. "ð—†ð–¾ð—â€”A web application tool and framework for the extensive comparison of metagenetic datasets." In Proceedings of Genome Informatics 2019. Cold Spring Harbor, 2019. [doi:10.7490/f1000research.1117645.1](https://doi.org/10.7490/f1000research.1117645.1).
>
> **[2017]**
>
> **[talk]** Molik, David, Michael Pfrender, Scott Emrich. â€œK-mer comparison methods in metagenomics, applications at the community level.â€ In Proceedings of Genome Informatics 2017. Cold Spring Harbor, 2017. [meetings.cshl.edu/abstracts.aspx?meet=INFO&year=17](https://meetings.cshl.edu/abstracts.aspx?meet=INFO&year=17). 
>
> **[2016]**
>
> **[poster]** Molik, David, Yu-Jui Ho, Ying Jin, and Molly Hammell. "Deployment of a Bioinformatics Analytics Platform in the Cloud." Proceedings of The Biological Data Science 2016 Cold Spring Harbor, 2016. [meetings.cshl.edu/abstracts.aspx?meet=DATA&year=16](https://meetings.cshl.edu/abstracts.aspx?meet=DATA&year=16).
> 
> **[talk]** Molik, David. â€œImplementing Best Practices in Bioinformatics, Issues of Data And Computational Tool Access That Will Affect Data Science in the Near Future.â€ In Data Analytics Across Disciplines: Academic Advancement And Business Competitiveness. National Professional Science Masterâ€™s Association, 2016. [www.dataanalyticsacrossdisciplines.org](http://www.dataanalyticsacrossdisciplines.org/).
>
>  **[poster]** Molik, David, Ying Jin, and Molly Hammell. â€œEDGYâ€”Export of Data from Galaxy to Yabi, Automated Workflow Transfer to Command Line Toolâ€ Proceedings of The Biology of Genomes, 2016. [meetings.cshl.edu/abstracts.aspx?meet=genome&year=16](https://meetings.cshl.edu/abstracts.aspx?meet=genome&year=16).
>
> **[2015]**
>
>  **[poster]** Molik, David, Ying Jin, and Molly Hammell. â€œAutomated Transfer of Workflows from Galaxy to Yabi and Command Line Tools.â€ In Proceedings of Genome Informatics 2015, 151. Cold Spring Harbor, 2015. [meetings.cshl.edu/abstracts.aspx?meet=info&year=15](https://meetings.cshl.edu/abstracts.aspx?meet=info&year=15).
> 
> **[talk]** Luciano, Joanne, Grant Cumming, Eva Kahana, Tara French, and David Molik. â€œHealth Web Science: Health Science, the Web and Web Observatories.â€ Proceedings of Informs Healthcare â€“ 2015. [meetings2.informs.org/wordpress/healthcare2015/](http://meetings2.informs.org/wordpress/healthcare2015/).
> 
>
> ## TEACHING
>
> Certified carpentries.org instructor, taught bioinformatics carpentries lessons for USDA ARS, three workshops in 2023, one workshop in 2024
>
> Designed and Taught University of Notre Dameâ€™s first Computational Genomics course, â€œBIOS 60132 - Introduction to Computational Genomicsâ€ a graduate level, three credit course teaching commonly used algorithms in Computational Genomics in Fall of 2020
>
> Organized and led in teaching â€œBioinformatics Introductory Program (BIP) Week 2020â€ a introductory bioinformatics program for incoming Life Sciences graduate students at the University of Notre Dame. Based on carpentries.org curriculum, the workshop spanned four days and had over 20 volunteers in Summer 2020, for Center for Digital Scholarship. Taught as a Lecturer in BIP Week 2021. 
>
> Taught Data Carpentries Genomics curriculum as a series of workshops over the course of Spring 2020, for Center for Digtial Scholarship. First in-person, and then online in Spring 2020, for Center for Digital Scholarship.
>
> Taught Data Carpentries Genomics curriculm as a series of workshops over the course of Spring 2020, for Center for Digtial Scholarship. First in-person, and then online in Spring 2020, for [Center for Digital Scholarship](https://cds.library.nd.edu/).
>
> Lecture Teaching assistant for "BIOS 10172 - Biology II: Molecules to Ecosystems" in Spring of 2019, for [Joesph O'Tousa](https://www3.nd.edu/~jotousa/labwww/).
>
> Lab Teaching assistant for "BIOS 31312 - General Ecology Laboratory" in Fall 2018, for [Dominic Chaloner](https://biology.nd.edu/people/dominic-chaloner/).
> 
> Lab Teaching assistant for "BIOS 21202 - General Biology B Laboratory" in Spring 2018, for [Dominic Chaloner](https://biology.nd.edu/people/dominic-chaloner/).
>
> ## ACHIEVEMENTS
>
> Undergraduate Thesis, titled: â€œOat: Comparing Ontologies and Textâ€ Focused on Semantic Web Technologies and Distance Measures, Advisor: [Deborah McGuinness](https://tw.rpi.edu/web/person/Deborah_L_McGuinness), Fall 2012
>
> Eagle Scout, Achieved the Rank of Eagle in the Boy Scouts of America
>
> ## Funding
>
> $150000, Co-PI, DoD Global Emering Infections Surveillance Grant. â€œUsing house flies as xenosurveillance tools to monitor existing, emerging and cryptic microbial threats to US militaryâ€
>
> Awarded 90000 in Oracle Cloud Credits for the build out of a scientific cloud application in 2021
>
> **AWS Grants** - 
> - Received 10000 in AWS Could Credits for Research for exploring alternative bioinformatics research systems in the Amazon Cloud in 2016 [Development of Open Source Software](https://aws.amazon.com/government-education/research-and-technical-computing/cloud-credit-for-research/previous-recipients)
> - Received 30000 in AWS Could Credits for Research for exploring alternative bioinformatics research systems in the Amazon Cloud for as co-investigator in 2017 with 
> - Received 20000 in AWS Could Credits for Research for building metagenomic enrichment databases pipelines in 2018, [Development of a Cloud Service](https://aws.amazon.com/government-education/research-and-technical-computing/cloud-credit-for-research/previous-recipients)
>
> Awarded "Rapid Exposure to Advanced Computational Training" (REACT) Program Grant, received $1500 in funding for attendance to Computational Short Course - Summer 2017
>
> Awarded â€œGraduate Student Union Conference Presentation Grantâ€ (GSU CPG) to attend CSHL Genome Informatics 2017
>
> Awarded "Graduate School Professional Development Award" (ND Graduate School Professional Development) Notebaert grant in the amount of $1000 to attend IEEE BIBM 2018 
>
> Awarded $1176.43 from University of Notre Dame Professional Development Zahm Research Travel Grant Fund for travel to CSHL Genome Informatics 2019
>
> Awarded â€œGraduate Student Union Conference Presentation Grantâ€ (GSU CPG) to attend Entomology 2019
>
> Awarded University of Notre Dame Professional Development Zahm Research Travel Grant Fund for attendance to CSHL Biological Data Science 2020
>
> **Society Memberships**
> - Society of Systematic Biologists
> - Sigma Xi

<meta name="keywords" content="David Molik, Molik, Curriculum Vitae, CV, Resume"/>
<meta name="description" content="David Molik's CV, information on past employment"/>
<meta name="subject" content="david.molik.co : A Personal Webpage">
<style>
table, th, td { border:0px solid black; }
img { border-width: 1px; border-color: #A9A9A9;}
a:link { color: 	#264073; }
a:visited { color: #264073; }
a:hover { color: #264073; }
a:active { color: #003399; }
hr {	background: #595959; }
blockquote { border-left: 1px solid #595959; }
code { color: #4d4d4d; }
p { color: #4d4d4d; } 
p.view  { display: none; }
</style>
