# Curriculum Vitae

[ [Research](/index.md) ][ **[Curriculum Vitae](/cv.md)** ][ [Github](https://github.com/molikd) ][ [Contact](/contact.md) ]

---
> # David C. Molik
> _Computational Biology PhD Student, University of Notre Dame_
>
> ## EDUCATION
>
> **University of Notre Dame, Notre Dame, IN**  
> Phd. Integrated BioMedical Sciences  
> Concentration in Computational Biology   
> Advisor: [Michael Pfrender](https://www3.nd.edu/~mpfrende/)    
> Aug/2016 â€“
>
> **Rensselaer Polytechnic Institute, Troy, NY**  
> BS. Information Technology and Web Science  
> Concentration in Communications  
> Aug/2009 â€“ Dec/2012  
> 
> ## EMPLOYMENT  
> 
> **Graduate Research Assistant in [Goodson lab](https://chemistry.nd.edu/people/holly-goodson/) , University of Notre Dame** â€“ Notre Dame, IN  
> May/2019 â€“ Aug/2019
> 
> **Graduate Research Assistant in [Emrich lab](http://web.eecs.utk.edu/~semrich/) , University of Notre Dame** â€“ Notre Dame, IN  
> Aug/2016 â€“ Dec/2017
>  
> **Scientific Informatics Developer, Cold Spring Harbor Labratory** â€“ Cold Spring Harbor,  NY  
> Dec/2014 â€“ Aug/2016  
>  
> **Software Engineer, Synacor** â€“ Buffalo, NY  
> July/2013 â€“ Nov/2014  
>  
> **Undergraduate Researcher, Tetherless World Constellation** â€“ Troy, NY  
> Aug/2012 â€“ Jan/2013  
>  
> **NSF REU, SUNY at Buffalo in [Corso Lab](http://web.eecs.umich.edu/~jjcorso/)** â€“  Buffalo, NY  
> May/2012 â€“ Aug/2012  
>  
> **Internship, Tetherless World Constellation** â€“ Troy, NY  
> May/2010 â€“ Aug/2010  
>  
> ## ARTICLES
> 
> æŽè¯£è¿œ *(Yiyuan Li)*, David Molik, Michael Pfrender. åŸºäºŽNextflowæž„å»ºçš„å®æ¡å½¢ç è‡ªåŠ¨åŒ–åˆ†æžæµç¨‹EPPSï¼ˆåŸºå› ç»„å¤šæ ·æ€§ä¸“åˆŠï¼‰*(EPPS, a metabarcoding bioinformatics pipeline using Nextflow)* . Biodiversity Science, 2019. [doi:10.17520/biods.2018211](http://www.biodiversity-science.net/CN/10.17520/biods.2018211).
> 
> Molik, David C., Michael Pfrender, Scott Emrich. "Effects from structure of Metabarcode Sequences on Lossy Analysis of Microbiome Data" IEEE International Conference on Bioinformatics and Biomedicine, Data Analytics in Metagenomics, 2018. [doi:10.1109/BIBM.2018.8621143](https://doi.org/10.1109/BIBM.2018.8621143).
>
> **[short]** Tomlinson, DeAndre. David Molik, Michael Pfrender, Scott Emrich. "The Effects of Normalization, Transformation, and Rarefaction on Clustering of OTU Abundance" IEEE International Conference on Bioinformatics and Biomedicine, 2018. [10.1109/BIBM.2018.8621347](https://doi.org/10.1109/BIBM.2018.8621347).
>
> Ho, Yu-Jui, Naishitha Anaparthy, David Molik, Grinu Mathew, Toby Aicher, Ami Patel, James Hicks, and Molly Gale Hammell. "Single-cell RNA-seq analysis identifies markers of resistance to targeted BRAF inhibitors in melanoma cell populations." Genome research, 2018, 1353-1363. [doi:10.1101/gr.234062.117](https://doi.org/10.1101/gr.234062.117). 
>
> Molik, David C., Michael Pfrender, Scott Emrich. "Methods in Description and Validation of Local Metagenetic Microbial Communities" Preprint: bioRxiv, 2017. [doi:10.1101/198614](https://doi.org/10.1101/198614).
>
> Cumming, Grant P., Tara French, Jamie Hogg, Douglas McKendrick, Heidi Gilstad, David Molik, and Joanne S. Luciano. "Trust and Provenance in Communication to eHealth Consumers." In The Digitization of Healthcare, pp. 189-203. Palgrave Macmillan, London, 2017. [doi:10.1057/978-1-349-95173-4_11](https://doi.org/10.1057/978-1-349-95173-4_11).
>
> Cumming, Grant P., Douglas McKendrick, Jamie Hogg, Tara French, Eva Kahana, David Molik, and Joanne S. Luciano. "Formulating eHealth Utilizing an Ecological Understanding." In The Digitization of Healthcare, pp. 81-98. Palgrave Macmillan, London, 2017. [doi:10.1057/978-1-349-95173-4_5](https://doi.org/10.1057/978-1-349-95173-4_5).
>
> Cumming, Grant P., Edward Morris, Paul Simpson, Tara French, Eva Kahana, Joanne S. Luciano, and David Molik. "The future of post-reproductive healthâ€“The role of the Internet, the Web, information provision and access." Post Reproductive Health, 2016, 123-130. [doi:10.1177/2053369116647858](https://doi.org/10.1177/2053369116647858).
> 
> **[short]** Molik, David, and Michelle Sharer. â€œThe Use of Recursive Bisection in the Mapping of Sample New York State Electoral Districts.â€ Proceedings of the 6th International Conference on Electronic Governance, 2012, 520â€“21. [doi:10.1145/2463728.2463844](https://doi.org/10.1145/2463728.2463844).
>
> ## ABSTRACTS - TALKS
> *Only Talks and Posters in which I am the First or Last Author*
>
> **[invited/talk]** Molik, David, Stephen Richards, Michael Pfrender. "Data mining for the prioritization of American insecta genomes." Entomology 2019. Entomological Society of America, 2019. https://esa.confex.com/esa/2019/meetingapp.cgi/Paper/143297.
>
> **[lightning talk/poster]** Molik, David. "ð—†ð–¾ð—â€”A web application tool and framework for the extensive comparison of metagenetic datasets." In Proceedings of Genome Informatics 2019. Cold Spring Harbor, 2019. [doi:10.7490/f1000research.1117645.1](https://doi.org/10.7490/f1000research.1117645.1).
>
> **[talk]** Molik, David, Michael Pfrender, Scott Emrich. â€œK-mer comparison methods in metagenomics, applications at the community level.â€ In Proceedings of Genome Informatics 2017. Cold Spring Harbor, 2017. https://meetings.cshl.edu/abstracts.aspx?meet=INFO&year=17 
>
> **[poster]** Molik, David, Yu-Jui Ho, Ying Jin, and Molly Hammell. "Deployment of a Bioinformatics Analytics Platform in the Cloud." Proceedings of The Biological Data Science 2016 Cold Spring Harbor, 2016. https://meetings.cshl.edu/abstracts.aspx?meet=DATA&year=16.
> 
> **[talk]** Molik, David. â€œImplementing Best Practices in Bioinformatics, Issues of Data And Computational Tool Access That Will Affect Data Science in the Near Future.â€ In Data Analytics Across Disciplines: Academic Advancement And Business Competitiveness. National Professional Science Masterâ€™s Association, 2016. http://www.dataanalyticsacrossdisciplines.org/.
>
>  **[poster]** Molik, David, Ying Jin, and Molly Hammell. â€œEDGYâ€”Export of Data from Galaxy to Yabi, Automated Workflow Transfer to Command Line Toolâ€ Proceedings of The Biology of Genomes, 2016. https://meetings.cshl.edu/abstracts.aspx?meet=genome&year=16.
>
>  **[poster]** Molik, David, Ying Jin, and Molly Hammell. â€œAutomated Transfer of Workflows from Galaxy to Yabi and Command Line Tools.â€ In Proceedings of Genome Informatics 2015, 151. Cold Spring Harbor, 2015. https://meetings.cshl.edu/abstracts.aspx?meet=info&year=15.
> 
> **[talk]** Luciano, Joanne, Grant Cumming, Eva Kahana, Tara French, and David Molik. â€œHealth Web Science: Health Science, the Web and Web Observatories.â€ Proceedings of Informs Healthcare â€“ 2015. http://meetings2.informs.org/wordpress/healthcare2015/.
>
> ## ACHIEVEMENTS
>
> Undergraduate Thesis, titled: â€œOat: Comparing Ontologies and Textâ€ Focused on Semantic Web Technologies and Distance Measures, Fall 2012
>
> **AWS Grants** - Received 10000 in AWS Could Credits for Research for exploring alternative bioinformatics research systems in the Amazon Cloud in 2016; received 30000 in AWS Could Credits for Research for exploring alternative bioinformatics research systems in the Amazon Cloud for as co-investigator in 2017; received 20000 in AWS Could Credits for Research for building metagenomic enrichment databases pipelines in 2019
>
> Awarded "Rapid Exposure to Advanced Computational Training" (REACT) Program Grant, received $1500 in funding for attendance to Computational Short Course - Summer 2017
>
> Awarded â€œGraduate Student Union Conference Presentation Grantâ€ (GSU CPG) to attend CSHL Genome Informatics 2017
>
> Awarded "Graduate School Professional Development Award" (ND Graduate School Professional Development) Notebaert grant in the amount of $1000 to attend IEEE BIBM 2018 
>
> Eagle Scout, Achieved the Rank of Eagle in the Boy Scouts of America

<meta name="keywords" content="David Molik, Molik, Curriculum Vitae, CV, Resume"/>
<meta name="description" content="David Molik's CV, information on past employment"/>
<meta name="subject" content="david.molik.co : A Personal Webpage">
<style>body {background-color: #cee4ae; color: #4d4d4d;}
img { border-width: 1px; border-color: #A9A9A9;}
a:link { color: 	#264073; }
a:visited { color: #264073; }
a:hover { color: #264073; }
a:active { color: #003399; }
hr {	background: #595959; }
blockquote { border-left: 1px solid #595959; }
code { color: #4d4d4d; }
p { color: #4d4d4d; } 
p.view  { display: none; }
</style>
