# Curriculum Vitae

[ [Research](/index.md) ][ **[Curriculum Vitae](/cv.md)** ][ [Github](https://github.com/molikd) ][ [Contact](/contact.md) ]

---
> # David C. Molik
> _Computational Biology PhD Student, University of Notre Dame_
>
> ## EDUCATION
>
> **University of Notre Dame, Notre Dame, IN**  
> Phd. Integrated BioMedical Sciences  
> Concentration in Computational Biology   
> Advisor: [Michael Pfrender](https://www3.nd.edu/~mpfrende/)    
> Aug/2016 –
>
> **Rensselaer Polytechnic Institute, Troy, NY**  
> BS. Information Technology and Web Science  
> Concentration in Communications  
> Aug/2009 – Dec/2012  
> 
> ## EMPLOYMENT  
> 
> **Graduate Scholar in [Navari Center for Digital Scholarship](https://cds.library.nd.edu/) , University of Notre Dame** – Notre Dame, IN  
> Jan/2020 – 
>
> **Graduate Research Assistant in collaboration with [Goodson lab](https://chemistry.nd.edu/people/holly-goodson/) , University of Notre Dame** – Notre Dame, IN  
> May/2019 – Aug/2019
> 
> **Graduate Research Assistant in collaboration with [Emrich lab](http://web.eecs.utk.edu/~semrich/) , University of Notre Dame** – Notre Dame, IN  
> Aug/2016 – Dec/2017
>  
> **Scientific Informatics Developer, [Cold Spring Harbor Labratory](https://www.cshl.edu/research/cancer/bioinformatics/)** – Cold Spring Harbor,  NY  
> Dec/2014 – Aug/2016  
>  
> **Software Engineer; Network Operations Center Manager, [Synacor](https://synacor.com/)** – Buffalo, NY  
> July/2013 – Nov/2014  
>  
> **Undergraduate Researcher, [Tetherless World Constellation](https://tw.rpi.edu//)** – Troy, NY  
> Aug/2012 – Jan/2013  
>  
> **NSF REU, SUNY at Buffalo in [Corso Lab](http://web.eecs.umich.edu/~jjcorso/)** –  Buffalo, NY  
> May/2012 – Aug/2012  
>  
> **Internship, [Tetherless World Constellation](https://tw.rpi.edu//)** – Troy, NY  
> May/2010 – Aug/2010  
>  
> ## ARTICLES
> 
> **[preprint, submitted]** David C. Molik, DeAndre Tomlinson, Shane Davitt, Eric L. Morgan, Benjamin Roche, Natalie Meyers, Michael E. Pfrender. "Combining Natural Language Processing and Metabarcoding to Reveal Pathogen-Environment Associations" bioRxiv, 2020. [doi:10.1101/2020.09.02.280578](https://doi.org/10.1101/2020.09.02.280578 )
>
> **[preprint, submitted]** Joseph Y Ong, Julia T Pence, David C Molik, Holly V Goodson. "Yeast grown in continuous culture systems can detect mutagens with improved sensitivity relative to the Ames test" bioRxiv, 2020. [doi:10.1101/2020.06.15.152371](https://doi.org/10.1101/2020.06.15.152371)
>
> Molik, David C., Michael Pfrender, Scott Emrich. “Uncovering Effects from the Structure of MetabarcodeSequences for Metagenetic and Microbiome Analysis” Methods and Protocols, 2020. [doi:10.3390/mps3010022](https://doi.org/10.3390/mps3010022)
>
> 李诣远 *(Yiyuan Li)*, David Molik, Michael Pfrender. 基于Nextflow构建的宏条形码自动化分析流程EPPS（基因组多样性专刊）*(EPPS, a metabarcoding bioinformatics pipeline using Nextflow)* . Biodiversity Science, 2019. [doi:10.17520/biods.2018211](http://www.biodiversity-science.net/CN/10.17520/biods.2018211).
> 
> Molik, David C., Michael Pfrender, Scott Emrich. "Effects from structure of Metabarcode Sequences on Lossy Analysis of Microbiome Data" IEEE International Conference on Bioinformatics and Biomedicine, Data Analytics in Metagenomics, 2018. [doi:10.1109/BIBM.2018.8621143](https://doi.org/10.1109/BIBM.2018.8621143).
>
> **[short]** Tomlinson, DeAndre. David Molik, Michael Pfrender, Scott Emrich. "The Effects of Normalization, Transformation, and Rarefaction on Clustering of OTU Abundance" IEEE International Conference on Bioinformatics and Biomedicine, 2018. [10.1109/BIBM.2018.8621347](https://doi.org/10.1109/BIBM.2018.8621347).
>
> Ho, Yu-Jui, Naishitha Anaparthy, David Molik, Grinu Mathew, Toby Aicher, Ami Patel, James Hicks, and Molly Gale Hammell. "Single-cell RNA-seq analysis identifies markers of resistance to targeted BRAF inhibitors in melanoma cell populations." Genome research, 2018, 1353-1363. [doi:10.1101/gr.234062.117](https://doi.org/10.1101/gr.234062.117). 
>
> Cumming, Grant P., Edward Morris, Paul Simpson, Tara French, Eva Kahana, Joanne S. Luciano, and David Molik. "The future of post-reproductive health–The role of the Internet, the Web, information provision and access." Post Reproductive Health, 2016, 123-130. [doi:10.1177/2053369116647858](https://doi.org/10.1177/2053369116647858).
> 
> **[short]** Molik, David, and Michelle Sharer. “The Use of Recursive Bisection in the Mapping of Sample New York State Electoral Districts.” Proceedings of the 6th International Conference on Electronic Governance, 2012, 520–21. [doi:10.1145/2463728.2463844](https://doi.org/10.1145/2463728.2463844).
>
> ## Chapters and Whitepapers
>
> RDA COVID-19 Working Group. recommendations and guidelines. Research Data Alliance. 2020. [doi:10.15497/rda00046](https://doi.org/10.15497/rda00046).
>
> Cumming, Grant P., Tara French, Jamie Hogg, Douglas McKendrick, Heidi Gilstad, David Molik, and Joanne S. Luciano. "Trust and Provenance in Communication to eHealth Consumers." In The Digitization of Healthcare, pp. 189-203. Palgrave Macmillan, London, 2017. [doi:10.1057/978-1-349-95173-4_11](https://doi.org/10.1057/978-1-349-95173-4_11).
>
> Cumming, Grant P., Douglas McKendrick, Jamie Hogg, Tara French, Eva Kahana, David Molik, and Joanne S. Luciano. "Formulating eHealth Utilizing an Ecological Understanding." In The Digitization of Healthcare, pp. 81-98. Palgrave Macmillan, London, 2017. [doi:10.1057/978-1-349-95173-4_5](https://doi.org/10.1057/978-1-349-95173-4_5).
>
> ## ABSTRACTS - TALKS
> *Only Talks and Posters in which I am the First or Last Author*
>
> **[poster]** Molik, David, DeAndre Tomlinson, Shane Davitt, Eric L. Morgan, Benjamin Roche, Natalie Meyers, Michael E. Pfrender. "Automated text categorization and metabarcoding reinforces Cryptococcus neoformans–woody decomposition association." Biological Data Science (Virtual). Cold Spring Harbor, 2020.
>
> **[invited/talk]** Molik, David, Stephen Richards, Michael Pfrender. "Data mining for the prioritization of American insecta genomes." Entomology 2019. Entomological Society of America, 2019. https://esa.confex.com/esa/2019/meetingapp.cgi/Paper/143297.
>
> **[lightning talk/poster]** Molik, David. "𝗆𝖾𝗍—A web application tool and framework for the extensive comparison of metagenetic datasets." In Proceedings of Genome Informatics 2019. Cold Spring Harbor, 2019. [doi:10.7490/f1000research.1117645.1](https://doi.org/10.7490/f1000research.1117645.1).
>
> **[talk]** Molik, David, Michael Pfrender, Scott Emrich. “K-mer comparison methods in metagenomics, applications at the community level.” In Proceedings of Genome Informatics 2017. Cold Spring Harbor, 2017. https://meetings.cshl.edu/abstracts.aspx?meet=INFO&year=17 
>
> **[poster]** Molik, David, Yu-Jui Ho, Ying Jin, and Molly Hammell. "Deployment of a Bioinformatics Analytics Platform in the Cloud." Proceedings of The Biological Data Science 2016 Cold Spring Harbor, 2016. https://meetings.cshl.edu/abstracts.aspx?meet=DATA&year=16.
> 
> **[talk]** Molik, David. “Implementing Best Practices in Bioinformatics, Issues of Data And Computational Tool Access That Will Affect Data Science in the Near Future.” In Data Analytics Across Disciplines: Academic Advancement And Business Competitiveness. National Professional Science Master’s Association, 2016. http://www.dataanalyticsacrossdisciplines.org/.
>
>  **[poster]** Molik, David, Ying Jin, and Molly Hammell. “EDGY—Export of Data from Galaxy to Yabi, Automated Workflow Transfer to Command Line Tool” Proceedings of The Biology of Genomes, 2016. https://meetings.cshl.edu/abstracts.aspx?meet=genome&year=16.
>
>  **[poster]** Molik, David, Ying Jin, and Molly Hammell. “Automated Transfer of Workflows from Galaxy to Yabi and Command Line Tools.” In Proceedings of Genome Informatics 2015, 151. Cold Spring Harbor, 2015. https://meetings.cshl.edu/abstracts.aspx?meet=info&year=15.
> 
> **[talk]** Luciano, Joanne, Grant Cumming, Eva Kahana, Tara French, and David Molik. “Health Web Science: Health Science, the Web and Web Observatories.” Proceedings of Informs Healthcare – 2015. http://meetings2.informs.org/wordpress/healthcare2015/.
>
> ## TEACHING
>
> Designed and Taught University of Notre Dame's first Computational Genomics course, "[BIOS 60132 - Introduction to Computational Genomics](http://david.molik.co/ND_ICG_2020/)" a graduate level, three credit course teaching commonly used algoithms in Computational Genomcs in Fall of 2020 
>
> Organized and Partially taught "Bioinformatics Introductory Program (BIP) Week 2020" a introductory bioinformatics program for incoming Life Sciences graduate students at the University of Notre Dame. Based on carpentries.org curriculum, the workshop spanned four days and had over 20 volunteers in Summer 2020, for [Center for Digital Scholarship](https://cds.library.nd.edu/).
>
> Taught Data Carpentries Genomics curriculm as a series of workshops over the course of Spring 2020, for Center for Digtial Scholarship. First in-person, and then online in Spring 2020, for [Center for Digital Scholarship](https://cds.library.nd.edu/).
>
> Lab Teaching assistant for "BIOS 30420/60562 - Aquatic Insects" in Fall 2019, for [Ronald Hellenthal](https://biodiversity.nd.edu/contact/).
> 
> Lecture Teaching assistant for "BIOS 10172 - Biology II: Molecules to Ecosystems" in Spring of 2019, for [Joesph O'Tousa](https://www3.nd.edu/~jotousa/labwww/).
>
> Lab Teaching assistant for "BIOS 31312 - General Ecology Laboratory" in Fall 2018, for [Dominic Chaloner](https://biology.nd.edu/people/dominic-chaloner/).
> 
> Lab Teaching assistant for "BIOS 21202 - General Biology B Laboratory" in Spring 2018, for [Dominic Chaloner](https://biology.nd.edu/people/dominic-chaloner/).
>
> ## ACHIEVEMENTS
>
> Undergraduate Thesis, titled: “Oat: Comparing Ontologies and Text” Focused on Semantic Web Technologies and Distance Measures, Advisor: [Deborah McGuinness](https://tw.rpi.edu/web/person/Deborah_L_McGuinness), Fall 2012
>
> Eagle Scout, Achieved the Rank of Eagle in the Boy Scouts of America
>
> ## Funding
>
> **AWS Grants** - Received 10000 in AWS Could Credits for Research for exploring alternative bioinformatics research systems in the Amazon Cloud in 2016; received 30000 in AWS Could Credits for Research for exploring alternative bioinformatics research systems in the Amazon Cloud for as co-investigator in 2017; received 20000 in AWS Could Credits for Research for building metagenomic enrichment databases pipelines in 2019
>
> Awarded "Rapid Exposure to Advanced Computational Training" (REACT) Program Grant, received $1500 in funding for attendance to Computational Short Course - Summer 2017
>
> Awarded “Graduate Student Union Conference Presentation Grant” (GSU CPG) to attend CSHL Genome Informatics 2017
>
> Awarded "Graduate School Professional Development Award" (ND Graduate School Professional Development) Notebaert grant in the amount of $1000 to attend IEEE BIBM 2018 
>
> Awarded $1176.43 from University of Notre Dame Professional Development Zahm Research Travel Grant Fund for travel to CSHL Genome Informatics 2019
>
> Awarded “Graduate Student Union Conference Presentation Grant” (GSU CPG) to attend Entomology 2019
>
> Awarded University of Notre Dame Professional Development Zahm Research Travel Grant Fund for attendance to CSHL Biological Data Science 2020
>

<meta name="keywords" content="David Molik, Molik, Curriculum Vitae, CV, Resume"/>
<meta name="description" content="David Molik's CV, information on past employment"/>
<meta name="subject" content="david.molik.co : A Personal Webpage">
<style>body {background-color: #cee4ae; color: #4d4d4d;}
img { border-width: 1px; border-color: #A9A9A9;}
a:link { color: 	#264073; }
a:visited { color: #264073; }
a:hover { color: #264073; }
a:active { color: #003399; }
hr {	background: #595959; }
blockquote { border-left: 1px solid #595959; }
code { color: #4d4d4d; }
p { color: #4d4d4d; } 
p.view  { display: none; }
</style>
